core/templates/dev/head/exploration_editor/ExplorationEditor.js

```diff
@@ -143,6 +146,17 @@ oppia.controller('ExplorationEditor', [
 
         explorationWarningsService.updateWarnings();
 
+        // Initialize changeList by the one received from the backend.
+        if (data.draft_changes) {
+          changeListService.loadAutosavedChangeList(data.draft_changes);
+        }
+        // Show modal displaying lost changes
```

Comment:
Comments should go as close to 80 chars as possible and then line break (also elsewhere).